terraform/environments/delius-core/README.md

```diff
@@ -1,91 +1,77 @@
 
-<!-- How people can raise an urgent issue with your service. This must not be the email address or phone number of an individual on your team, it should be a shared email address, phone number, or website that allows someone with an urgent issue to raise it quickly. -->
+To ensure that people looking at your runbook can get the information they need quickly, your runbook should be short but clear. Throughout, only use acronyms if you’re confident that someone who has just been woken up at 3am would understand them. -->
 
-### **Service team contact:**
+  
 
-<!-- How people with non-urgent issues or questions can get in touch with your team. As with incident contact details, this must not be the email address or phone number of an individual on the team, it should be a shared email address or a ticket tracking system.-->
+_If you have any questions surrounding this page please post in the `#ask-probation-hosting` channel._
 
-### **Hosting environment:**
+  
 
-Modernisation Platform
+## Mandatory Information
 
-<!-- If your service is hosted on another MOJ team’s infrastructure, link to their runbook. If your service has another arrangement or runs its own infrastructure, you should list the supplier of that infrastructure (ideally linking to your account’s login page) and describe, simply and briefly, how to raise an issue with them. -->
+  
 
-## Optional
+### **Last review date:**
 
-### **Other URLs:**
+  17/12/24
 
-<!--  If you can, provide links to the service’s monitoring dashboard(s), health checks, documentation (ideally describing how to run/work with the service), and main GitHub repository. -->
+<!-- Adding the last date this page was reviewed, with any accompanying information -->
 
-### **Expected speed and frequency of releases:**
+  
 
-<!-- How often are you able to release changes to your service, and how long do those changes take? -->
+### **Description:**
+This directory contains the Terraform configuration for the Delius Core environments, which are primarily comprised of the following resources:
 
-### **Automatic alerts:**
+* WebLogic app ECS
+* WebLogic EIS ECS
+	* external interface services
+	* serves API endpoints to allow services to interact with Delius data
+* LDAP ECS
+	* used for authenticating users in Delius, MIS, NextCloud, User Management, PWM, and apps in Cloud Platform
+* Password manager (PWM) ECS
+* Oracle EC2 databases
+	* used by Weblogic, GDPR (now in CP)
+* Alfresco EFS + SFS
+	* Alfresco is mainly hosted in Cloud Platform but some resources remain in MP
 
-<!-- List, briefly, problems (or types of problem) that will automatically alert your team when they occur. -->
 
-### **Impact of an outage:**
+<!-- A short (less than 50 word) description of what your service does, and who it’s for.-->
 
-<!-- A short description of the risks if your service is down for an extended period of time. -->
+  
 
-### **Out of hours response types:**
+### **Service URLs:**
```

Comment:
Not sure if it's worth templating these URLs just so it's obvious there's more than just the dev environment.

Prod URL format will differ, but we can update that later once prod is built.